.github/actions/setup/action.yml

```diff
@@ -11,6 +11,22 @@ inputs:
 runs:
   using: composite
   steps:
+      - name: Verify args
```

Comment:
Instead of adding these steps here, we might consider putting them in a new action in `trinodb/github-actions` repository. This would allow avoiding adding the complicated expression in `ref` to every checkout action in `ci.yml`.

Reply:
@hashhar you were against this before, WDYT?